Bookstore.cpp

```diff
@@ -0,0 +1,89 @@
+#include<iostream>
+#include<cstring>
+using namespace std;
+
+#include"Bookstore.h"
+
+Bookstore::Bookstore()
+{
+    for(int i = 0; i < 4; i++)
+    	books[i] = NULL;
+    bookCount = 0;
+}
+
+void Bookstore::addBook()
+{
+	cout << "Enter Book Title, Price, Quantity";
+	int price, quantity, i;
+	bool index = -1;
+	char title[10];
+	cin >> title >> price >> quantity;
+	for(i = 0; i < 4; i++) {
+		if(NULL == books[i]) {
+			if(-1 != index)
+				index = i;
+		}
+		else if(0 == strcmp(title, books[i]->getBookTitle())) {
+            books[i]->setBookStock(books[i]->getBookStock() + quantity);
+            cout << "Book with the title already Exist. Updated stocks"<<endl;
+            return;
+		}
+	}
+	if(-1 == index) {
```

Comment:
Add Null pointer and array index out of bound exception - maruti

Reply:
This Handling is not required - chetan